site/app/templates/polls/PollForm.twig

```diff
@@ -202,7 +222,20 @@
         }
         submitErrorChecks();
     };
+    window.onload = function() {
+        toggleTimerInputs();
+    };
```

Comment:
Is this needed now?  I would be better to render everything on the server side when the page initially loads rather than relying on client-side rendering to render the initial state of the page.

Reply:
Changed to the server side.